modules/system/effect-wfrp4e.js

```diff
@@ -168,6 +168,7 @@ export default class ActiveEffectWFRP4e extends WarhammerActiveEffect
 
     get radius()
     {
+        if (typeof this.applicationData.radius === 'number') return this.applicationData.radius;
```

Comment:
if radius is a number - parsing it as roll was throwing an exception for me, but again - maybe i was doing somethig wrong?

Reply:
This looks like it's from before the library integration, so it'll need to be updated.

Reply:
looks like it, i will remove that.